### ISSUE TITLE
Adding a new entry gives a messed up layout

### ISSUE THREAD

**Issue by ioogithub** [keeweb/keeweb#1356]
**Describe the bug**
I have imported a database that I made in Keepass, it works perfectly in KeepassXC and Keepass2Android. When I try to create a new entry I am given a messed up layout, the data fields are overlapping with the titles and misalligned

**To Reproduce**
1. Open existing database.kdbx in KeeWeb 
2. Observe that everything looks correct.
3. Click on + to add a new entry of type "ID Card (template).
4. The form that is displayed is quit messed up (see screenshot).

**Expected behavior**
This app is advertised as being "compatible with KeePass". This is a standard databased that has been used in Keepass, KeepassX, KeepassXC, Keepass2Android, KeePassDroid without any issues. I would expect that if KeeWeb is Keepass compatible it would be able to handle it correctly.

**Screenshots**
![keeweb](https://user-images.githubusercontent.com/58017942/69487943-f8aaa580-0e5a-11ea-8a70-450eef13698e.png)

**Environment**
Your user-agent (from Settings/Help section)

**Kdbx File**
Does it happen on Demo or New database?  
If you have a test db without your personal data, please attach it.

**Logs**
Please open dev tools in your browser and attach output log from Console tab. If you are using a desktop app, devtools can be opened from Settings/General/Advanced.

**Comment by antelle** [comment]
Hi! You have some plugin in your database that adds these "_etm_*" fields.

**Comment by ioogithub** [comment]
I don't believe it is a plug in but rather these are the default templates that the Android client "Keepass2Android" create.

I recently tested opening the database with the other Keepass clients (Keepass and KeepassXC for example) and they render and handle the attribute fields properly.
